.eslintrc

```
@@ -1,9 +1,10 @@
 {
   "parser": "babel-eslint",
   "extends": ["airbnb-base", "plugin:prettier/recommended", "prettier"],
```

Let's add the jest/recommended rule set.
```suggestion
  "extends": ["airbnb-base", "plugin:jest/recommended", "plugin:prettier/recommended", "prettier"],
```